### PR TITLE
Upgrade build process to Babel 7

### PR DESCRIPTION
**Pull request by mwegman**
[SPARK-63292](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-63292) #done

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-rooms/src/rooms.js [hunk 1]
```diff
@@ -435,7 +435,7 @@ const Rooms = WebexPlugin.extend({
   getRoomEvent(webex, activity, event) {
     try {
       const sdkEvent = cloneDeep(this.eventEnvelope);
-      const cluster = getHydraClusterString(webex, activity.object.url);
+      const cluster = getHydraClusterString(webex, activity.url);
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-rooms/src/rooms.js:438]: Where did this change come from?

Comment (mwegman) [reply packages/node_modules/@webex/plugin-rooms/src/rooms.js:438]: I'm guessing the convo team wanted to remove "unnecessary" properties. I'm getting a `url` property from a place I know won't go away.

**Review by adamweeks — APPROVED** [review]
Awesome!

**Comment by taymoork2** [comment]
Excuse me what?????
https://media.giphy.com/media/KWbmr5E1UdPUI/giphy.gif

**Pull request merged**